tests/landIce/FO_GIS/CMakeLists.txt

```diff
@@ -88,28 +87,27 @@ ENDIF()
 
 set (testName ${testNameRoot}_Unstructured)
 
-if (ALBANY_EPETRA)
-  # First run
-  configure_file(${CMAKE_CURRENT_SOURCE_DIR}/input_fo_gis_unstruct.yaml
-                 ${CMAKE_CURRENT_BINARY_DIR}/input_fo_gis_unstruct.yaml)
-  add_test(${testName}_Epetra ${Albany.exe} input_fo_gis_unstruct.yaml)
-  set_tests_properties(${testName}_Epetra
-                       PROPERTIES
-                       LABELS            "LandIce;Epetra;Forward"
-                       FIXTURES_REQUIRED PopulateMeshes
-                       FIXTURES_SETUP    UnstructRestart)
 
-  if (NOT ALBANY_DISABLE_OUTPUT_MESH)
-    # Restart
-    configure_file(${CMAKE_CURRENT_SOURCE_DIR}/input_fo_gis_unstruct_restart.yaml
-                   ${CMAKE_CURRENT_BINARY_DIR}/input_fo_gis_unstruct_restart.yaml)
-    add_test(${testName}_Restart ${Albany.exe} input_fo_gis_unstruct_restart.yaml)
-    set_tests_properties(${testName}_Restart
-                         PROPERTIES
-                         LABELS "LandIce;Epetra;Forward"
-                         RUN_SERIAL TRUE
-                         FIXTURES_REQUIRED UnstructRestart)
-  endif()
+# First run
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/input_fo_gis_unstruct.yaml
```

Comment:
I think the unstruct and unstructT tests are now very similar (if not the same). Can we prune one of them?

Reply:
Yes, I'll remove one.